### PR TITLE
Fix caching blobs, avoid misc PHP notices

### PR DESCRIPTION
**Pull request by miken32**
Under certain circumstances, these notices can stop reloads from working.

### CODE REVIEW

**Comment by tm1000** [comment]
You need to set val AND the query. The way you did it the truncated value is stored in the database and not the uuid

File: amp_conf/htdocs/admin/libraries/BMO/DB_Helper.class.php [hunk 1]
```diff
@@ -321,7 +321,7 @@ public function setConfig($key = null, $val = false, $id = "noid") {
 				$uuid = $this->insertBlob(null, $query[':val'], $query[':type']);
 			}
 			$query[':type'] = "blob";
-			$query[':val'] = $uuid;
+			$val = $uuid;
```
Comment (tm1000) [inline amp_conf/htdocs/admin/libraries/BMO/DB_Helper.class.php]: You need to set val AND the query. The way you did it the truncated value is stored in the database and not the uuid

Comment (miken32) [reply amp_conf/htdocs/admin/libraries/BMO/DB_Helper.class.php]: Yes, that makes sense; in all my testing the code exited before hitting the statement execution but I can see where that might not always be the case.

**Comment by tm1000** [comment]
I broke this up into 14 and 15

- https://github.com/FreePBX/framework/commit/78328252f629280629f185aac4b9cff656fb07cc
- https://github.com/FreePBX/framework/commit/9d73018f5823f5a44c23a10a0eebe2c8572640fc

**Pull request closed without merging**